crates/uv/src/commands/pip_install.rs

```diff
@@ -161,6 +160,7 @@ pub(crate) async fn pip_install(
     if reinstall.is_none()
         && upgrade.is_none()
         && source_trees.is_empty()
+        && overrides.is_empty()
```

Comment:
I realized that `satisfies` doesn't respect overrides at all, and it's actually difficult to support in our current model.